### PR TITLE
Add mail_from option and changed help text

### PR DESCRIPTION
**Pull request by jbrunink**
Made some changes to the e-mail backup function so you can specify from what e-mail the e-mail is being sent from. Also, made the help text for use_tls and use_ssl more clear. (thanks to Django documentation)

Could you take a look at this PR?  Thank you!

Best regards, Jurgen

### CODE REVIEW

File: dsmr_backend/migrations/0009_auto_20200103_1230.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.9 on 2020-01-03 11:30
```
Comment (dennissiemensma) [inline dsmr_backend/migrations/0009_auto_20200103_1230.py]: You might want to rename this file to something more verbose like `0009_emailsettings_email_from.py`.

Don't forget to also rename the record in the Django migrations table, in your local database afterwards.

**Comment by dennissiemensma** [comment]
Thanks! It's merged and I've updated the changelog now as well.

**Comment by dennissiemensma** [comment]
Released in v2.13.0

**Comment by dennissiemensma** [comment]
Reminder for next time to automatically fill in the new field with the existing value. See #827.

I didn't think about it as well. I might create a migration for it.

**Comment by jbrunink** [comment]
Oops, my fault. Thought about it yesterday and had already a PR ready, but migration is the way to go.

**Pull request merged**